warehouse/query-core/src/test/java/datawave/query/iterator/QueryIteratorIT.java

```diff
@@ -557,17 +557,6 @@ public void event_leadingRegex_shardRange_secondEvent_test() throws IOException
         event_test(seekRange, query, false, null, addEvent(11, "123.345.457"), Arrays.asList(getBaseExpectedEvent("123.345.457")));
     }
     
-    @Test(expected = IOException.class)
```

Comment:
Why is this test being removed? Is there still a way to test an invalid config? For example, what if every configured ivarator directory is invalid? (instead of hdfs://blah it's hddfs://blah?). In that case everything should prune to zero, and we should get an appropriate failure message.

Reply:
Added back unit test and modified it to fail when the config is invalid.